src/Billing/Controllers/FreshdeskController.cs

```diff
@@ -81,11 +82,11 @@ public class FreshdeskController : Controller
                     note += $"<li>Org, {orgNote}</li>";
                     if (!customFields.Any(kvp => kvp.Key == "cf_org"))
```

Comment:
I believe in this if we could use the `_billingSettings.FreshDesk.OrgFieldName` instead of the hardcoded string

Reply:
@aj-rosado Nice catch, thank you!